app/src/main/cpp/src/appMain/CMakeLists.txt

```diff
@@ -152,6 +153,8 @@ endif()
 
 list(APPEND LIBRARIES bson -L${BSON_LIBS_DIRECTORY})
 list(APPEND LIBRARIES emhashmap -L${EMHASHMAP_LIBS_DIRECTORY})
+list(APPEND LIBRARIES crypto -L${OPENSSL_LIBS_DIRECTORY})
+list(APPEND LIBRARIES ssl -L${OPENSSL_LIBS_DIRECTORY})
```

Comment:
@LitvinenkoIra please make sure that it is no an Android specific changes and will work on all builds

Reply:
I have a successful build for Linux on my side (0ba626a commit + changes from https://github.com/CustomSDL/sdl_core/pull/279)